typescript/packages/jumble/src/recipes/index.ts

```diff
@@ -0,0 +1,6 @@
+import smolIframeSrc from "./smolIframe.tsx?raw";
+import smolIframe from "./smolIframe.js";
+import { addRecipe } from "@commontools/runner";
+
+addRecipe(smolIframe, smolIframeSrc);
+export { smolIframe };
```

Comment:
When iterating on charms this recipe has to exist, otherwise you can't actually call iterate.